canonical-websites/snapcraft.io
# Show blank graphs and message when lacking data.

# Done

- Implemented some styling for no data
- Added notification at the top of the page
- Made sure axis still show on graphs without data

# QA

- Pull the branch
- `./run`
- Visit http://0.0.0.0:8004/account/snaps/SNAP_NAME/measure where SNAP_NAME is the name of a snap without any data
- Everything in 'Done' should be there.

# Screenshot

![screenshot-2018-3-14 publisher metrics for lukewh-test-snap 3](https://user-images.githubusercontent.com/479384/37417257-b970a824-27a7-11e8-9c81-c2d8a51d1c7c.png)
